gelf/message_test.go

```diff
@@ -27,6 +29,14 @@ func TestInvalidFieldNames(t *testing.T) {
 	testMessage.Send()
 }
 
+func TestTimestamp(t *testing.T) {
+	now := float64(time.Now().Unix())
+	testMessage := NewMessage(LevelInfo, "This is a short test message.", "This is a long test message.")
+	if math.Abs(testMessage.Timestamp-now) > 1 {
```

Comment:
That's kind of a funny way to make sure it's in seconds -- you could just check bounds on the thing and make sure it's not a factor of 1e9 too big.